storage/client_test.go

```diff
@@ -260,7 +261,8 @@ func TestGetObjectEmulated(t *testing.T) {
 		if err := w.Close(); err != nil {
 			t.Fatalf("closing object: %v", err)
 		}
-		got, err := client.GetObject(context.Background(), bucket, want.Name, -1, nil, nil)
+		const defaultGen = int64(-1)
```

Comment:
I wasn't very specific about this, but perhaps this should be a "global" constant at the top of a file? Perhaps in `storage.go` as well? I can see a few other files (`writer.go`, `storage_test.go` to name a few) where it might be useful, but no need to change any of those now.